### PR TITLE
fix: Signup from org wizard invitation and other tweaks [CAL-1973] [CAL-1993] [CAL-1994]

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Fixes a few things:

1. Removed "Disable Cal.com branding" and "Book a Team Member" options from organization appearance settings section.

1. Organization wizard admin invitation sign-up screen shows correct domain and locks username
<kbd>image</kbd>

2. NIT that affected my OCD, arrow next to username in a non-org context to show user dropdown had no separation
<kbd></kbd>

3. Corrected the logo selector when creating a team, was showing a person, now shows what we show for selecting an organization avatar
<kbd></kbd>

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

See screenshots above and description.

## Mandatory Tasks
- [X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by ericrommel — COMMENTED** [review]
@leog, regarding item 'Removed "Disable Cal.com branding" and "Book a Team Member" options from organization appearance settings section', should they be also applied to organization teams?
image

**Comment by leog** [comment]
> @leog, regarding item 'Removed "Disable Cal.com branding" and "Book a Team Member" options from organization appearance settings section', should they be also applied to organization teams?

Yes, teams should be able to have those options, we only removed them from organizations as they are not really a conventional team, have different treatment when it comes to its profile which is what the appearance settings apply to.

**Comment by leog** [comment]
> @leog, regarding item 'Removed "Disable Cal.com branding" and "Book a Team Member" options from organization appearance settings section', should they be also applied to organization teams?

@ericrommel yes, teams should be able to have those options, we only removed them from organizations as they are not really a conventional team, have different treatment when it comes to its profile which is what the appearance settings apply to.

File: apps/web/pages/signup.tsx [hunk 1]
```diff
@@ -246,6 +271,8 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
         email: verificationToken.identifier,
         username,
       },
+      orgSlug:
+        (orgInfo?.organization && (orgInfo?.organization.slug || userOrgMetadata?.requestedSlug)) ?? null,
```
Comment (alannnc) [inline apps/web/pages/signup.tsx:275]:
```suggestion
      orgSlug:
        orgInfo?.organization?.slug || userOrgMetadata?.requestedSlug ?? null,
```
We can simplify first sentence

**Review by alannnc — CHANGES_REQUESTED** [review]
Left a NIT comment, after that should be Good.

**Comment by ericrommel** [comment]
What do you mean by "...and locks username"? Can it be changed by the invited user?

**Comment by leog** [comment]
> What do you mean by "...and locks username"? Can it be changed by the invited user?

@ericrommel it can’t be changed by the user; we want to strictly follow using their email user, so that eric@acme.com becomes acme.cal.com/eric

**Comment by Jaibles** [comment]
> @ericrommel it can’t be changed by the user; we want to strictly follow using their email user, so that [eric@acme.com](mailto:eric@acme.com) becomes acme.cal.com/eric

I know that we want follow their email so that it's easy to turn 10 or 100 emails into user accounts, but does it have to be strict in that the user cannot change it? cc @PeerRich

**Review by ericrommel — APPROVED** [review]
Everything looks fine.

**Pull request merged**